projects/idleDao.js

```diff
@@ -0,0 +1,61 @@
+
+const sdk = require('@defillama/sdk');
```

Comment:
this only counts the tokens on polygon, whereas most idle tvl is on ethereum. the current idle adapter has already been updated to have polygon